src/posts/paper/13-q-and-a.ts

```diff
@@ -218,8 +218,12 @@ Additionally, we have seen [data](https://pubs.rsna.org/doi/full/10.1148/ryct.20
 As a result, getting a vaccination yourself confers a reduction in incoming microCOVIDs (as outlined [above](#1-how-much-less-likely-is-it-for-vaccinated-individuals-to-catch-covid-reduction-in-microcovids)),
 but we do not recommend increasing your microCOVID budget on top of that given current data. This may change as more data about the vaccines is gathered.
 
-If you do decide you want to increase your microCOVID budget, discuss this with any regular close contacts you have, such as housemates, partners, or co-workers - your increased risk of COVID, asymptomatic or not, could affect them more than it affects you. 
-Also consider the potential impact on your community at large.
+If you do decide you want to increase your microCOVID budget post vaccination,
+* Discuss this with any regular close contacts you have, such as housemates, partners, or co-workers — your increased risk of COVID, asymptomatic or not, could affect them more than it affects you.
+* Consider the potential impact on your community at large — taking on more risk of being an asymptomatic spreader could result in you infecting many other people.
+
+### So is a vaccinated person safer to be around than an unvaccinated person?
+Not necessarily. A person's risk is the product of their precautions and activities. Just as a person who wears a mask to indoor parties is not safer than a person who doesn't wear a mask but also doesn't leave their house, a person who is vaccinated but does 5x more risky behavior is not safer than an unvaccinated person who is staying isolated. To accurately understand a person's risk, you need to model what behaviors the person is doing, including the MODifier for being vaccinated.
```

Comment:
This helps to add this. I wanna try a re-write of it.

> Not necessarily. A person's risk is the product of their precautions and activities. It is true that a vaccinated person gets the modifier in the table listed above for each of the activities they do. If a vaccinated person and a non-vaccinated person did the exact same activities over 23 days, the vaccinated person would be 1/5 as risky (with Modern/Pfizer) for you to be around (or about 1/2 as risky with AstraZeneca). However, if someone gets vaccinated and then starts doing more risky behaviors, they would increase the risk they pose to you. The only way to accurately understand their risk is to model their activities, total their score, and multiplier it by the modifier from the table above. (The Risk Tracker(LINKED) makes it easier to do this modeling.)

Thoughts?

Also, I don't think this is the place to use "MODifier" ... it's just "modifier"

Reply:
Settling on your language but with numbers pulled out to reduce mental load.